testapps/test_pillowtop/tests/test_case_pillow.py

```diff
@@ -34,6 +34,8 @@ def setUp(self):
             self.elasticsearch = get_es_new()
             initialize_index_and_mapping(self.elasticsearch, CASE_INDEX_INFO)
             initialize_index_and_mapping(self.elasticsearch, CASE_SEARCH_INDEX_INFO)
+        delete_es_index(CASE_INDEX_INFO.index)
+        delete_es_index(CASE_SEARCH_INDEX_INFO.index)
```

Comment:
wait doesn't this need to come before the `initialize_index_and_mapping` calls?

Reply:
Weird. I wonder how that was working before